tests/func-tests/common/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  AlchemyProvider,
+  JsonRpcProvider,
+  TransactionReceipt,
+  TransactionResponse,
+} from '@ethersproject/providers';
+
+// todo: move to common
```

Comment:
isn't this already in common? is this comment still useful?

Reply:
Good catch! Fixed.